### DISCUSSION TITLE
Hyperlink for item row in List

### DISCUSSION THREAD

**Discussion by Tech-Dex** [fyne-io/fyne#4041] [Q&A] [answered]
I see that you are able to select an item row from the list, is there any chance to interact with the "click" in order to set a hyperlink?

<img width="796" alt="image" src="https://github.com/fyne-io/fyne/assets/44111988/1fe884db-2aab-48d1-afa6-deb4ff0312ce">

**Comment by andydotxyz** [comment, marked as answer]
You can put any code in the `List.OnSelected` callback and from there do anything you want...

**Reply by Tech-Dex** [reply]
Sorry for the late response.
Yes that works great.

```
	providerDealListW.OnSelected = func(id int) {
		deals, _ := providerDealListB.Get()
		dLink := providersDealListUrls[deals[id]]

		var storeProductUrl *url.URL
		storeProductUrl, _ = url.Parse(dLink)
		
		fyne.CurrentApp().OpenURL(storeProductUrl)
	}
```

I am not sure if this is the best option to Open a URL, but I check hyperlink.go and found that's the way that it is done. If you have a better solution that is already built-in list I will appreciate it.

**Reply by andydotxyz** [reply]
Well, I guess the builtin way is just to use the `Hyperlink` widget. With the benefit it actually looks like a link.
